src/components/LoadingIcon/PLoadingIcon.vue

```diff
@@ -1,19 +1,38 @@
 <template>
-  <div class="p-loading-icon">
+  <div class="p-loading-icon" :class="iconSizeClass">
     <div class="p-loading-icon__segment" />
     <div class="p-loading-icon__segment" />
     <div class="p-loading-icon__segment" />
     <div class="p-loading-icon__segment" />
   </div>
 </template>
 
+<script setup lang="ts">
+  import { computed, defineProps } from 'vue'
+
+  const props = defineProps<{
+    size?: 'small' | 'large',
+  }>()
```

Comment:
should we be consistent here and just add an optional `small` prop? I don't feel strongly, just crossed my mind.

Reply:
Up to you, this ultimately doesn't matter much other than just improving the devx/consistency. We could never change it and probably wouldn't matter lol.

Reply:
Yeah don't feel like you need to address this. 

I don't really have a preference. Individual props are kinda nice at implementation, but they can get weird in component code. Either is fine by me, so if we do unify these maybe we just pick whichever one is already used the most so we have the least amount of changes to make.